draft-ietf-dnsop-dns-catalog-zones.md

```diff
@@ -373,10 +380,12 @@ ignored.  If a broken catalog zone was transferred, the newly transferred
 catalog zone MUST be ignored (but the older copy of the catalog zone SHOULD
 be left running subject to values in SOA fields).
 
+[FIXME: reword this if we do coo, or the CSYNC immediate flag, or something else]
 If there is a clash between an existing member zone's name and an incoming
 member zone's name (via transfer or update), the new instance of the zone MUST
 be ignored and an error SHOULD be logged.
 
+[FIXME: all this ordering of operation might be an undue burden on implementations. What's the harm in a secondary failing to get a member zone immediately, and then getting it successfully a minute later?]
```

Comment:
Indeed, the order of events for adding or deleting a zone is not something we should  anchor in this draft. I'm in favor of leaving this entirely to the implementation/operator.

Reply:
I agree!